### PR TITLE
Feature/centralize docs

### PR DESCRIPTION
**Pull request by creisle**
- Copy documents from IPR python and GraphKB python repos to here (centralize documentation
- Create script to generate field descriptions from IPR python json spec
- Include IPR client screenshots to some field pages

Note: This must be merged AFTER the corresponding IPR python adaptor release is tagged

https://github.com/bcgsc/pori_ipr_python/pull/38

All of the docs/graphkb files were directly copied and have not been modified

### CODE REVIEW

File: docs/graphkb/matching/gene.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+This will use a similar algorithm to what we have seen above in the disease matching example.
+
+![initial graph](../images/pori-gene-matching-1.png)
+
+In the graph above the relationship types shown are: `GeneralizationOf` (G), `ElementOf` (E),
+`DeprecatedBy` (D), and `CrossReferenceOf` (X).
+
+## Match by Name
+
+Like before, the first thing done is to match the input name
```
Comment (elewis2) [inline docs/graphkb/matching/gene.md:21]: would possibly use 'As' instead of 'Like'

Comment (creisle) [reply docs/graphkb/matching/gene.md]: ya that sounds better. changed!

File: docs/graphkb/matching/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+subgraph of $G$ containing only vertices of the class type specified in the query (ex. Disease)
+be, $V_t$.
+
+All edges between these vertices are then categorized into two disjoint sets: synonym-like
+($E_{syn}$) or inheritance-like ($E_{inh}$). By default the synonym-like edges are:
+GeneralizationOf, AliasOf, CrossReferenceOf, DeprecatedBy, and Infers. Whereas the
+inheritance-like edges are: SubClassOf, and ElementOf.
+
+!!! Note "Edge Groups are Configurable"
+    These are the default division of Edges. However, classes used the edge sets
```
Comment (elewis2) [inline docs/graphkb/matching/index.md:20]: do you mean 'used for' or 'using'?

Comment (creisle) [reply docs/graphkb/matching/index.md:20]: yup! the former

**Review by elewis2 — APPROVED** [review]

**Pull request merged**